monitoring/uptime/uptime.go

```diff
@@ -0,0 +1,157 @@
+// Package uptime contains samples of uptime checks.
+package uptime
+
+import (
+	"fmt"
+	"io"
+	"log"
+
+	monitoring "cloud.google.com/go/monitoring/apiv3"
+	"github.com/golang/protobuf/ptypes/duration"
+	"golang.org/x/net/context"
+	"google.golang.org/api/iterator"
+	"google.golang.org/genproto/googleapis/api/monitoredres"
+	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
+)
+
+// [START monitoring_uptime_check_create]
+
+// create creates an example uptime check.
+func create(w io.Writer, projectID string) {
+	ctx := context.Background()
+	client, err := monitoring.NewUptimeCheckClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Close()
+	req := &monitoringpb.CreateUptimeCheckConfigRequest{
+		Parent: "projects/" + projectID,
+		UptimeCheckConfig: &monitoringpb.UptimeCheckConfig{
+			DisplayName: "new uptime check",
+			Resource: &monitoringpb.UptimeCheckConfig_MonitoredResource{
+				MonitoredResource: &monitoredres.MonitoredResource{
+					Type: "uptime_url",
+					Labels: map[string]string{
+						"host": "example.com",
+					},
+				},
+			},
+			CheckRequestType: &monitoringpb.UptimeCheckConfig_HttpCheck_{
+				HttpCheck: &monitoringpb.UptimeCheckConfig_HttpCheck{
+					Path: "/",
+					Port: 80,
+				},
+			},
+			Timeout: &duration.Duration{Seconds: 10},
+			Period:  &duration.Duration{Seconds: 300},
+		},
+	}
+	uc, err := client.CreateUptimeCheckConfig(ctx, req)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Fprintf(w, "Successfully created uptime check %q\n", uc.GetDisplayName())
+}
+
+// [END monitoring_uptime_check_create]
+
+// [START monitoring_uptime_check_list_configs]
+
+// list is an example of listing the uptime checks in projectID.
```

Comment:
This comment doesn't add anything to the documentation line "To list your current uptime checks, do the following:"

Reply:
Agreed. I like to have valid GoDoc, though. So, I think it's OK.